### PR TITLE
[Mgmt Track 2] 5156 generate empty resource autorest change

### PR DESCRIPTION
**Pull request by allenjzhang**

### CODE REVIEW

**Comment by allenjzhang** [comment]
We have a problem here. Since we keyed on PUT for the resource model, both compute and storage had different creation model than the actual model. That had forced me to add renaming and causing conflict. This would create problems in our unified API model. Let's discuss in .NET sync.

File: samples/Azure.Management.Storage/Generated/Configuration.json [hunk 1]
```diff
@@ -1 +1,36 @@
-{"OutputFolder":".","Namespace":"Azure.Management.Storage","LibraryName":null,"SharedSourceFolders":["..\\..\\..\\artifacts\\bin\\AutoRest.CSharp\\Debug\\netcoreapp3.0\\Generator.Shared","..\\..\\..\\artifacts\\bin\\AutoRest.CSharp\\Debug\\netcoreapp3.0\\Azure.Core.Shared"],"AzureArm":true,"PublicClients":true,"ModelNamespace":true,"HeadAsBoolean":true,"SkipCSProjPackageReference":true,"CredentialTypes":[],"CredentialScopes":[],"CredentialHeaderName":"api-key","OperationGroupToResourceType":{"Operations":"Microsoft.Storage/operations","Skus":"Microsoft.Storage/skus","Usages":"Microsoft.Storage/locations/usages","PrivateLinkResources":"Microsoft.Storage/storageAccounts/privateLinkResources"},"OperationGroupToResource":{"Operations":"Operation","Skus":"Sku","Usages":"Usage","PrivateLinkResources":"PrivateLinkResource"},"ResourceRename":{"BlobServiceProperties":"BlobService","FileServiceProperties":"FileService","StorageAccountCreateParameters":"StorageAccount"}}
\ No newline at end of file
+{
```
Comment (m-nash) [inline samples/Azure.Management.Storage/Generated/Configuration.json:1]: This will fail code checks.  This is a generated file and it won't match what the regeneration does

File: samples/Azure.ResourceManager.Sample/Generated/Configuration.json [hunk 2]
```diff
@@ -1 +1,42 @@
...
\ No newline at end of file
+{
```
Comment (m-nash) [inline samples/Azure.ResourceManager.Sample/Generated/Configuration.json:1]: same comment as above

File: src/AutoRest.CSharp/AutoRest.CSharp.csproj [hunk 3]
```diff
@@ -61,15 +61,15 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <_GeneratedPropsFileLines Include="&lt;Project>" />
```
Comment (m-nash) [inline src/AutoRest.CSharp/AutoRest.CSharp.csproj:64]: why is this needed?

**Review by m-nash — CHANGES_REQUESTED** [review]

File: src/AutoRest.CSharp/AutoRest.CSharp.csproj (hunk 3 above)
Comment (allenjzhang) [reply src/AutoRest.CSharp/AutoRest.CSharp.csproj:64]: The changes were done by VS for proper XML escaping formatting. Let's keep it as other folks's save probably would have trigger this later anyway.

**Review by m-nash — APPROVED** [review]
